test/built-ins/Array/fromAsync/asyncitems-null-undefined.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-array.fromasync
+description: >
+  Array.fromAsync rejects with a TypeError if the asyncItems argument is null or undefined
+info: |
+  3.c. Let usingAsyncIterator be ? GetMethod(asyncItems, @@asyncIterator).
```

Comment:
Fine with not having the chapter and verse, but if it's present then I believe the place the error is thrown is:
```suggestion
  3.k.ii. Let arrayLike be ! ToObject(asyncItems).
```
(and I believe that ! should be a ? in the proposal)

Reply:
Up to you. On a second look it was actually pretty obvious, so I must have just had a blind spot for it on the previous review.